addons/web/static/src/legacy/scss/res_partner.scss

```diff
@@ -0,0 +1,29 @@
+.o_kanban_view.o_res_partner_kanban {
```

Comment:
Shouldn't the css be in base, wiht the view definition ?

Reply:
I actually hesitated about that... But I saw that the base module barely had any css files :smile: